example/eblob_backend.cpp

```diff
@@ -1528,3 +1540,54 @@ int blob_send_new(struct eblob_backend_config *c, void *state, struct dnet_cmd *
 	         dnet_print_error(err));
 	return err;
 }
+
+int blob_bulk_read_new(struct eblob_backend_config *c, void *state, struct dnet_cmd *cmd, void *data,
+		       struct dnet_cmd_stats *cmd_stats) {
+	using namespace ioremap::elliptics;
+
+	if (c == nullptr || state == nullptr || cmd == nullptr || data == nullptr)
+		return -EINVAL;
+
+	auto st = reinterpret_cast<dnet_net_state *>(state);
+	const int backend_id = c->data.stat_id;
+	auto backend_io = dnet_get_backend_io(st->n->io, backend_id);
+	if (!backend_io) {
+		DNET_LOG_ERROR(c->blog, "EBLOB: {}: couldn't find backend_io for backend_id: {}",
+			       __func__, backend_id);
+		return -EINVAL;
+	}
+
+	dnet_bulk_read_request bulk_request;
+	deserialize(data_pointer::from_raw(data, cmd->size), bulk_request);
+
+	int err = 0;
+	dnet_read_request request;
+	request.ioflags = bulk_request.ioflags;
+	request.read_flags = bulk_request.read_flags;
+	request.data_offset = request.data_size = 0;
+	request.deadline = bulk_request.deadline;
+
+	cmd->flags &= ~DNET_FLAGS_NEED_ACK;
+	const auto num_keys = bulk_request.keys.size();
+	for (size_t i = 0; i < num_keys; ++i) {
+		struct dnet_cmd cmd_copy(*cmd);
+		cmd_copy.id = bulk_request.keys[i];
+
+		auto read_stats = *cmd_stats;
+
+		const bool last_read = i >= (num_keys - 1);
+		{
+			dnet_oplock_guard oplock_guard{backend_io, &cmd_copy.id};
+			err = blob_read_new_impl(c, state, &cmd_copy, &read_stats, request, last_read);
+		}
+		if (err) {
+			cmd_copy.status = err;
+			dnet_send_reply(st, &cmd_copy, nullptr, 0, last_read ? 0 : 1);
+		}
+		cmd_copy.cmd = DNET_CMD_READ_NEW;
+		dnet_backend_command_stats_update(nullptr, backend_io, &cmd_copy, read_stats.size, 0, err,
+						  read_stats.handle_time);
```

Comment:
`read_stats.handle_time` will be zero here since `cmd_stats->handle_time` is zero and `blob_read_new_impl()` doesn't fill it. This time should be measured here.